sonic_platform_base/sfp_base.py

```diff
@@ -0,0 +1,266 @@
+
+        Returns:
+            An integer number of supply voltage in mV
+        """
+        return NotImplementedError
+    
+    def get_tx_bias(self):
+        """
+        Retrieves the TX bias current of this SFP
+
+        Returns:
+            A list of four integer numbers, representing TX bias in mA
+            for channel 0 to channel 4.
+            Ex. ['110.09', '111.12', '108.21', '112.09']
+        """
+        return NotImplementedError
+    
+    def get_rx_power(self):
+        """
+        Retrieves the received optical power for this SFP
+
+        Returns:
+            A list of four integer numbers, representing received optical
+            power in mW for channel 0 to channel 4.
+            Ex. ['1.77', '1.71', '1.68', '1.70']
+        """
+        return NotImplementedError
+    
+    def get_tx_power(self):
+        """
+        Retrieves the TX power of this SFP
+
+        Returns:
+            A list of four integer numbers, representing TX power in mW
+            for channel 0 to channel 4.
+            Ex. ['1.86', '1.86', '1.86', '1.86']
+        """
+        return NotImplementedError
+    
+    def reset(self):
+        """
+        Reset SFP and return all user module settings to their default srate.
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        return NotImplementedError
+    
+    def tx_disable(self, tx_disable):
+        """
+        Disable SFP TX for all channels
+
+        Args:
+            tx_disable : A Boolean, True to enable tx_disable mode, False to disable
+                         tx_disable mode.
+
+        Returns:
+            A boolean, True if tx_disable is set successfully, False if not
+        """
+        return NotImplementedError
+    
+    def tx_disable_channel(self, channel, disable):
+        """
+        Sets the tx_disable for specified SFP channels
+
+        Args:
+            channel : A hex of 4 bits (bit 0 to bit 3) which represent channel 0 to 3,
+                      e.g. 0x5 for channel 0 and channel 2.
+            disable : A boolean, True to disable TX channels specified in channel,
+                      False to enable
+
+        Returns:
+            A boolean, True if successful, False if not
+        """
+        return NotImplementedError
+    
+    def set_lpmode(self, lpmode):
+        """
+        Sets the lpmode (low power mode) of SFP
+
+        Args:
+            lpmode: A Boolean, True to enable lpmode, False to disable it
+            Note  : lpmode can be overridden by set_power_override
+
+        Returns:
+            A boolean, True if lpmode is set successfully, False if not
+        """
+        return NotImplementedError
+    
+    def set_power_override(self, power_override, power_set):
+        """
+        Sets SFP power level using power_override and power_set
+
+        Args:
+            power_override : 
+                    A Boolean, True to override set_lpmode and have power_set
```

Comment:
Change "have power_set" to "use power_set"

Reply:
Changed on Friday 03/29.